extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/InfinispanClientProcessor.java

```diff
@@ -148,7 +143,10 @@ InfinispanPropertiesBuildItem setup(ApplicationArchivesBuildItem applicationArch
 
             InfinispanClientProducer.handleProtoStreamRequirements(properties);
             Collection<ClassInfo> initializerClasses = index.getAllKnownImplementors(DotName.createSimple(
-                    SerializationContextInitializer.class.getName()));
+                    GeneratedSchema.class.getName()));
+            initializerClasses
```

Comment:
Seems this code is unneeded? It is the same as just above.

Reply:
Yes should be `SerializationContextInitializer` on the first line. On my tests using GeneratedSchema did not work and I don't want to break people still using SerializationContextInitializer